src/Magnum/LibOvrIntegration/Hmd.h

```diff
@@ -337,6 +337,15 @@ class MAGNUM_LIBOVRINTEGRATION_EXPORT Hmd {
             return ++_frameIndex;
         }
 
+        /**
+         * @brief Enable/Disable the performance HUD
+         *
+         * Performance HUD enables the HMD user to see information critical to
+         * the real-time operation of the VR application such as latency timing
+         * and CPU & GPU performance metrics.
+         */
+        void setPerformanceHudMode(const PerformanceHudMode& mode) const;
```

Comment:
This is just an four-byte enum, right? I'd pass it directly by value, passing it by reference costs 8 bytes (twice as much). Other than that it's ready for merge :)